src/services/decoders/ZstdDecoder/index.ts

```diff
@@ -0,0 +1,15 @@
+import {ZSTDDecoder} from "zstddec";
```

Comment:
for the issue discussed at https://github.com/y-scope/yscope-log-viewer/pull/368/files#r2301199409 , can we try again with

```suggestion
import {ZSTDDecoder} from "zstddec/stream";
```